src/app/src/main/res/layout/activity_accept_requests.xml

```diff
@@ -6,4 +6,38 @@
     android:layout_height="match_parent"
     tools:context=".AcceptRequestsActivity">
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/choose_location"
+            android:textSize="24sp" />
+
+        <com.google.android.gms.maps.MapView
+            android:id="@+id/map"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
+            android:layout_weight="1" />
+
+<!--        <fragment xmlns:android="http://schemas.android.com/apk/res/android"-->
+<!--            android:id="@+id/map"-->
+<!--            android:layout_width="match_parent"-->
+<!--            android:layout_height="0dp"-->
+<!--            android:layout_weight="1"-->
+<!--            class="com.google.android.gms.maps.SupportMapFragment" />-->
```

Comment:
Can these be removed?

Reply:
Yup you right, I didn't realize that got left in